Voycar.Api.Web/Features/Members/Endpoints/Get/Personal/Mapper.cs

```diff
@@ -0,0 +1,37 @@
+namespace Voycar.Api.Web.Features.Members.Endpoints.Get.Personal;
+
+using Entities;
+using Plans.Repository;
+using Users.Repository;
+
+
+public class Mapper : ResponseMapper<Response, Member>
+{
+    private readonly IPlans _planRepository;
+
+
+    public Mapper(IPlans planRepository, IUsers userRepository)
```

Comment:
```suggestion
    public Mapper(IPlans planRepository)
```

Das User Repository wird hier nicht benutzt